cascade-cli/src/Cascade/CLI.hs

```diff
@@ -0,0 +1,99 @@
+module Cascade.CLI (main) where
+
+import qualified Cascade.Api
+import qualified Data.Pool                          as Pool
+import           Data.Pool                           ( Pool
+                                                     , createPool
+                                                     )
+import qualified Database.PostgreSQL.Simple         as Postgres
+import           Data.Version                        ( Version
+                                                     , showVersion
+                                                     )
+import qualified Paths_cascade_cli                  as Meta
+                                                     ( version )
+import           Options.Applicative                 ( Parser
+                                                     , ParserInfo
+                                                     , execParser
+                                                     , fullDesc
+                                                     , help
+                                                     , helper
+                                                     , info
+                                                     , infoOption
+                                                     , long
+                                                     , progDesc
+                                                     , short
+                                                     , showDefault
+                                                     , value
+                                                     , metavar
+                                                     , auto
+                                                     , option
+                                                     , strOption
+                                                     )
+import           Development.GitRev                  ( gitCommitDate
+                                                     , gitHash
+                                                     )
+
+mkDatabaseConnectionPool :: PostgresOptions -> IO (Pool Postgres.Connection)
+mkDatabaseConnectionPool PostgresOptions {..} = do
+  createPool acquire Postgres.close 1 10 10
+ where
+  connectionInfo = Postgres.ConnectInfo { connectHost     = postgresHost
+                                        , connectPort     = postgresPort
+                                        , connectUser     = postgresUser
+                                        , connectPassword = postgresPassword
+                                        , connectDatabase = postgresDatabase
+                                        }
+  acquire = Postgres.connect connectionInfo
+
+runCascadeApi :: Options -> IO ()
+runCascadeApi Options {..} = do
+  databaseConnectionPool <- mkDatabaseConnectionPool postgresOptions
+  Cascade.Api.main <| Cascade.Api.Config { port = httpPort, withDatabaseConnection = Pool.withResource databaseConnectionPool }
+
+cascadeVersion :: Version -> String
```

Comment:
There is no need to get `Version` as an input. We can simply use `Meta.version`.